### PR TITLE
Feature/scan unifier

### PR DESCRIPTION
**Pull request by benmaidel**
Moved cob_scan_unifier from cob_navigation to cob_drivers

related PRs:
ipa320/cob_robots#364
ipa320/ipa_navigation#84
ipa320/cob_navigation#56

### CODE REVIEW

**Comment by benmaidel** [comment]
tested on cob4-2: scan unifier starts as expected

**Comment by mgruhler** [comment]
LGTM

**Comment by fmessmer** [comment]
Fine with me.

Just three minor comments:
Within the last 4 commits, the scan_unifier files get added (https://github.com/ipa-bnm/cob_driver/commit/70b40296853b60b7d879f057afa7404f52180d0a), then removed again (https://github.com/ipa-bnm/cob_driver/commit/aff911d42af5f88943c6a565fb01a99bdd202eac), then added again (https://github.com/ipa-bnm/cob_driver/commit/e7e13d4a8df73b5f4b2d28c427a294168416f867)... Does this always happen when moving part of a repo with history?

Could you possibly post the commands for moving part of a repo with history (or a link to a tutorial where you found the commands)? For documentation....

Currently the other cob_driver packages have version `0.6.5` while cob_scan_unifier still has `0.6.3`. I think it won't be a problem, I would just like to get a confirmation....

File: cob_scan_unifier/package.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>cob_scan_unifier</name>
+  <version>0.6.3</version>
+  <description>The cob_scan_unifier package holds code to unify two or more laser-scans to one unified scan-message</description>
+
+  <maintainer email="frm@ipa.fraunhofer.de">Florian Mirus</maintainer>
```
Comment (fmessmer) [inline cob_scan_unifier/package.xml]: This should be changed. Who takes over?

Comment (benmaidel) [inline cob_scan_unifier/package.xml]: i can take over!

**Comment by benmaidel** [comment]
@ipa-fxm: the way i have done it results in this kind of adding and removing. Don't know if there is a better way.

moving directories from one to another repo:

```
cd RepositoryA
git remote rm origin
git filter-branch --subdirectory-filter ContentA -- --all
# everithing from the directory "ContentA" is now in the root dir of the repositoryA
mkdir ContentA
mv * ContentA

git add ContentA
git commit

# Do not push anything! Change to your Destinations Repository (RepositoryB)
cd RepositoryB
git remote add Repository-A ../RepositoryA
git pull Repository-A master
# Enter your Merge Commit Message and delete the temporary remote
git remote rm Repository-A
```

look here http://stuermer-benjamin.de/programming/moving-files-slash-directories-between-git-repositories-without-loosing-history/

**Comment by fmessmer** [comment]
Formal Review OK
(Critical Review not relevant as this PR only moves already functioning code!)

**Pull request merged**